metricbeat/vendor/vendor.json

```diff
@@ -2,12 +2,6 @@
 	"comment": "",
 	"ignore": "test github.com/elastic/beats",
 	"package": [
-		{
```

Comment:
@tsg It is still strange that there are new LICENSE files for `go-sysinfo` for example and it does not show up as a new commit id in the vendor.json? Same for the other 4 where licene files were added.

Reply:
Uhm, yes, I manually copied the license files in some cases 😇 

In some cases, govendor doesn't copy those files, because they are at a different level than the go files. That was the case of `go-ole/go-ole`. I'm thinking manually copying it is the best approach for that case.

For `go-sysinfo` and `go-windows` I only copied them that way because the upstream PRs that were adding the licenses upstream weren't merged yet. I'll do those the right way now.

Reply:
Hmm, I tried to upgrade the two but they both come with a few code changes unrelated to the license. Fine if we do the upgrade in a new PR so that the code changes are easier to review?

Reply:
@tsg yes

For the go-ole one I worry that every time we update we forget the LICENSE but not sure how to fix that.

Reply:
We can't forget because the script will now return an error if any dep doesn't have the license.